### DISCUSSION TITLE
Breathe…

### DISCUSSION THREAD

**Discussion by DemianNernst** [share121/inter-knot#1901] [其他]
![The Dark Side of the Moon_109951168014530296](https://github.com/user-attachments/assets/a8c778c1-6e1a-4ed4-9d8b-5cbbfd3cd96f)
Breathe, breathe in the air,
Don't be afraid to care.
Leave, but don't leave me,
Look around choose your own ground.
For long you live and high you fly,
And smiles you'll give and tears you'll cry,
And all you touch and all you see,
Is all your life will ever be.
Run, rabbit run,
Dig that hole, forget the sun.
And when at last the work is done,
Don't sit down it's time to dig another one.
For long you live and high you fly,
But only if you ride the tide,
And balanced on the biggest wave,
You race towards an early grave.

**Comment by cpu644** [comment]
吗的前摇打过来了？不过月之暗面确实神专

**Comment by hfwang132** [comment]
最符合绝区零唱片店的一集
